src/main/java/ua/com/javarush/oleksandr/reddit/redditcloneabstract/model/User.java

```diff
@@ -1,27 +1,37 @@
 package ua.com.javarush.oleksandr.reddit.redditcloneabstract.model;
 
-import jakarta.persistence.Entity;
-import jakarta.persistence.GeneratedValue;
-import jakarta.persistence.GenerationType;
-import jakarta.persistence.Column;
-import jakarta.persistence.Id;
+import jakarta.persistence.*;
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
 import lombok.NoArgsConstructor;
+import org.hibernate.annotations.CreationTimestamp;
 
+import java.time.ZonedDateTime;
+
+@Entity
+@Table(name = "\"user\"")
 @Data
 @AllArgsConstructor
 @NoArgsConstructor
 @Builder(builderMethodName = "with")
-@Entity
 public class User {
+
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     @Column(name = "id")
     private Long userId;
+
     private String email;
```

Comment:
Going forward, we can add some constraints and an index to the email field if it is deemed appropriate.